gpep517/scheme.py

```diff
@@ -0,0 +1,154 @@
+# SPDX-FileCopyrightText:
+# SPDX-License-Identifier: MIT
+
+# Most of this file is copied from https://github.com/pypa/installer/
+# with minor tweaks where classes don't have hooks to control their
+# internals.
+
+from __future__ import annotations
+
+import contextlib, io, os, shlex, zipfile
+import typing as T
```

Comment:
Inconsistent.

Reply:
Again, all this does is make files longer and more annoying to read, since most type annotations suddenly gain an additional 5 characters of overhead towards the max line length.

Either that or you spend more time managing individual `from typing import Foo, Bar, Baz` imports than you spend coding.